### DISCUSSION TITLE
Html not redering correctly in iframe after livewire updates

### DISCUSSION THREAD

**Discussion by jochemgruter** [livewire/livewire#3523] [7. Bugs (archived)] [answered]
Since version v2.5.2 some code in my project has broken. I think it has something to do with PR #3152.

My component:

```
public function render()
{
    $preview = view('shop.shop')->render();

    return view('shop.preview', ['preview' => $preview]);
}
```

My view:
```
<iframe class="w-full h-full" srcdoc="{{$preview}}"></iframe>
```

When the page load, the iframe works fine. But when the component updates, it break. The content op srcdoc when livewire updates is only
```
<!DOCTYPE html>
<html lang=
```

Seems like it breaks with the first quote.

In version 2.5.1 everything works fine, but with 2.5.2 it doesn't

**Comment by ryatkins** [comment]
What is the purpose of the iframe?  

Have you tried wrapping the iframe in a div?

```
<div>
<iframe></iframe
</div>
```

**Comment by joshhanley** [comment, marked as answer]
@jochemgruter there was a bug in 2.5.2. Can you please upgrade to 2.5.5 and test it again, and let us know if it's working or not?

**Reply by jochemgruter** [reply]
Thank you. Worked!

**Reply by joshhanley** [reply]
No worries 🙂

**Comment by ryatkins** [comment]
@jochemgruter  I’m curious to the use case of an iframe, what’s the purpose?

**Reply by jochemgruter** [reply]
Its to preview a template that the user can customize. With the iframe I am able to show the responsive and the normal layout.
